session10/src/Homework/Ex8.java

```diff
@@ -0,0 +1,20 @@
+package Homework;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+
+public class Ex8 {
+    public static void main(String[] args) {
+        ArrayList<String> movies = new ArrayList<>(Arrays.asList("Cloudy", "Tormented", "The Godfather"));
+        Collections.sort(movies);
```

Comment:
ok, but sorting is not required to search in a list (neither `ArrayList` or `LinkedList`), **only arrays** require sorting before searching.